cmd/utils.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"github.com/conduktor/ctl/schema"
+	"github.com/spf13/cobra"
+	"strconv"
+)
+
+// used to detect when flag wasn't used
+const unprobableInt = -23871
```

Comment:
it this cobra standard ?

Reply:
Not at all lol.
It's honestly a quick hack. The issue I have is that for the optional flag, I want to detect when a user enters a value or I get the default value (I figured out that the default 0 could be a desired value by a user).
What I would really need, in fact, is a cobra function that returns a pointer on int/string instead of int/string with null when no value is there. I didn't found it but honestly didn't look really hard for that.

Reply:
Found something clean